src/components/product/ProductSummary/ProductSummary.tsx

```diff
@@ -30,7 +31,7 @@ function ProductSummary({ product }: Props) {
     [spotPrice, offers]
   )
 
-  const linkProps = useProductLink(product)
+  const linkProps = useProductLink({ ...product, index })
```

Comment:
This is not possible because of the relation between Props and product:

```interface Props {
  product: ProductSummary_ProductFragment
  index: number
} ```